File: frontend/src/app/model/payment.ts
export interface Payment {
  id: string;
  streetId: string;
  houseId: string;
  paymentDate: string;
  amount: number;
  paymentConceptId: string;
  paymentSubConceptId: string;
  paymentMode: string;
  notes: string;
  validated: boolean;
  files: FileList;
}

File: backend/src/main/java/org/uresti/pozarreal/dto/Payment.java
package org.uresti.pozarreal.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.format.annotation.DateTimeFormat;

import java.time.LocalDate;

@Builder
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Payment {
    private String id;
    private String houseId;
    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    private LocalDate paymentDate;
    @DateTimeFormat(iso = DateTimeFormat.ISO.DATE)
    private LocalDate registrationDate;
    private String userId;
    private Double amount;
    private String paymentConceptId;
    private String paymentSubConceptId;
    private String paymentMode;
    private String notes;
    private boolean validated;
    private boolean conflict;
}